config/routes.rb

```diff
@@ -26,4 +25,5 @@
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+  get '/:category', to: 'categories#show', param: :name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.